.gitignore

```diff
@@ -17,4 +17,5 @@ drivers/
 /docs/api.json
 .android-sdk/
 .gradle/
-nohup.out
\ No newline at end of file
+nohup.out
+api.json
```

Comment:
Why this change? Do we now generate into <root>/api.json? If so, remove the line 17?